lib/components/collapse.vue

```diff
@@ -43,10 +43,9 @@
         },
 
         watch: {
-            visible(newVal, oldVal){
-                if (newVal !== this.show) {
-                    this.toggle()
-                }
+            visible(newVal) {
+                this.show = newVal;
```

Comment:
Yep :D It can be just swapped with this.show = ... . Also we need emitting `change` event on emitState